ci/e2e-tests/helper-functions.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# This file contains several helper functions. These functions expect that
+# certain env vars are already set including $suite_id and $AZURE_RESOURCE_GROUP_NAME.
+
+# Create an IoT Hub instance.
+# Parameters:
+#   $1 - IoT hub name
+function createHub {
```

Comment:
All cases of `function foo {` should be `foo() {`. The former is ksh syntax that bash has for ksh-compatibility.

Reply:
Done.